### PR TITLE
Add release notes for v0.9.1

### PR DESCRIPTION
**Pull request by Jaanki**
Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

**Review by SteveMattar — APPROVED** [review]

**Review by mkolesnik — APPROVED** [review]

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1
+
+* Fixed --kubecontext in subctl commands and now it properly works as it used to.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Fixed --kubecontext in `subctl` commands and now it properly works as it used to.
```

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ title = "Releases"
...
+* Simplified `subctl cloud prepare aws` to extract credentials, infra id and region from local configuration field (if available).
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]:
```suggestion
* Simplified `subctl cloud prepare aws` to extract credentials, infra id and region from local configuration file (if available).
```

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ title = "Releases"
...
+* Fixed support to set natt-discovery-port and udp-port via node annotations.
```
Comment (nyechiel) [inline src/content/community/releases/_index.en.md]: What do you mean "fixed support"?

**Review by nyechiel — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md]:
```suggestion
* The `--kubecontext` flag in `subctl` commands now works properly.
```

File: src/content/community/releases/_index.en.md (hunk 2 above)
Comment (tpantelis) [reply src/content/community/releases/_index.en.md]:
```suggestion
* Simplified `subctl cloud prepare aws` to extract the credentials, infra id and region from a local configuration file (if available).
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1
+
+* The `--kubecontext` flag in `subctl` commands now works properly.
+* Simplified `subctl cloud prepare aws` to extract the credentials, infrastructure ID, and region from a local configuration file (if available).
+* natt-discovery-port and udp-port can now be set via node annotations.
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]:
```suggestion
* The `natt-discovery-port` and `udp-port` options can now be set via node annotations.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Review by tpantelis — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Pull request merged**